builtin/google/cloudrun/deployment.go

```diff
@@ -171,7 +171,7 @@ func (d *Deployment) pollServiceReady(
 				return service, nil
 
 			case "False":
-				return nil, fmt.Errorf("service failed to get ready")
+				return nil, fmt.Errorf("Service failed to get ready; check GCR logs for troubleshooting info.")
```

Comment:
GCR is Google Container Registry - I think "Cloud Run" might be better here.

I also hesitate a bit to punt them to the underlying platform here - it seems like the problem could just be that their app is crashing, and their app logs in waypoint could be the most useful thing.

Reply:
In this case there is no app in Waypoint to check, because the service didn't start. The deploy never succeeded.

Reply:
How about scrap that line and just specify Cloud Run at the start, to clarify which service exactly failed to get ready: `Cloud Run service failed to get ready`

Reply:
Oh, got it! Maybe now that we have status checks, it would make sense not to block the deploy on this check, so they can see those logs in waypoint? Not related to this change though 👍